spruce-skill/__sidebar.md

```diff
@@ -0,0 +1,17 @@
+-   [Building skills](/spruce-skill/)
```

Comment:
Why the extra underscore in the filename?  I'm not sure this will work w/ docsify.  Have you run `yarn docs` and verified it's working locally?